css/index.css

```diff
@@ -92,26 +98,19 @@ h2{
 */
 /* CSS for container divs for page content*/
 .container {
+    /*Display*/
+    display: flex;
+    align-items: center;
+    flex-direction: column;
     /* Setting width of .container elements */
-    width: 90vw;
     /* Setting the max width of the main content as 1024px*/
-    max-width: 1024px;
-    /* Centering the main content relative to the viewport*/
-    margin: 0 auto;
 }
 
 .inner-container{
     width: 100%;
     height: auto;
 }
 
-/* CSS for container divs for navigation bars */
-.nav-container {
-    /* Display */
-    display: flex;
-    justify-content: space-between;
-}
-
 .heading-container{
```

Comment:
I just noticed that the only styling we need for `heading-container` is centering the text, so we can simplify this style down to just this: 
```
.heading-container {
  text-align: center;
}
```
I might rename the css class to just `header` or something like that as well since classes with the name container in them are typicall divs that hold other html elements.

Reply:
Thank you for your comments regarding my work so far! ☺. I will make sure to follow your advices regarding centering the text. 

You are right about the css class name, I will make sure to fix that as well!